src/applications/static-pages/representative-status/components/App/States/Auth.jsx

```diff
@@ -1,38 +1,208 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
+import RepresentativeStatusApi from '../../../api/RepresentativeStatusApi';
+import { parsePhoneNumber } from '../../../utilities/phoneNumbers';
 
-export const Auth = ({ hasRepresentative, DynamicHeader }) => {
-  return (
-    <>
-      {hasRepresentative ? null : (
-        <va-card>
-          <div className="auth-no-rep-card">
-            <div className="auth-no-rep-header-icon">
-              <va-icon
-                icon="account_circle"
-                size={4}
-                srtext="add some text for a screen reader to describe the icon's semantic meaning"
-              />{' '}
-            </div>
-            <div className="auth-no-rep-text">
-              <DynamicHeader className="auth-no-rep-header">
-                You don’t have an accredited representative
-              </DynamicHeader>
-              <div className="auth-no-rep-body">
+export const Auth = ({ DynamicHeader, DynamicSubheader }) => {
+  const [representative, setRepresentative] = useState({});
+
+  const fetchRepStatus = async () => {
```

Comment:
What are your thoughts on moving the network call outside of the component? I think the component's responsibility should only be to render the information for a provided representative prop or render the empty state